cpt-dl/src/cptdl/utilities.py

```diff
@@ -7,7 +7,9 @@
 
 def read_dlauth():
     if not (Path.home().absolute() / '.pycpt_dlauth').is_file():
-        print('You need to set up an IRIDLAUTH! Please set up an IRI account here (https://iridl.ldeo.columbia.edu/auth/signup), then use cptdl.setup_dlauth("your_iri_login_email") to setup the cookie. This only needs to happen once!  ')
+        print('You need to set up an IRIDLAUTH! Please set up an IRI account here (https://iridl.ldeo.columbia.edu/auth/signup), then use cptdl.setup_dlauth("your_iri_login_email")')
+        print('It is possible that after gaining access, you will need to generate your ID from https://iridl.ldeo.columbia.edu/auth/genkey. Please copy the JSON obtained into your ".pycpt_dlauth" file to setup the cookie.')
+        print('This only needs to happen once!  ')
```

Comment:
Did you try following the original instructions before you replaced them? It looks like `setup_dlauth` gets an API key and creates `~/.pycpt_dlauth` for you.